pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AutoConsumeSchema.java

```diff
@@ -308,13 +311,26 @@ public void fetchSchemaIfNeeded(SchemaVersion schemaVersion) throws SchemaSerial
         }
     }
 
+    private SchemaVersion getSchemaVersion(byte[] schemaVersion) {
```

Comment:
static ?

Reply:
Does the static method have some advantage?

Reply:
Probably not a big deal from a performance perspective.
But if a method does not use state from the instance it is better to mark it 'static'.

Reply:
Ok, I'll fix this.